packages/create-plugin/templates/provisioning-template/provisioning/dashboards/providers.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: 1
+
+providers:
+  # <string> an unique provider name. Required
+  - name: put-plugin-id-here
```

Comment:
since these templates are rendered as templates, we can easily put the plugin id here.

Reply:
+1